services/metadata/frontend/content/Category.svelte

```diff
@@ -41,7 +41,7 @@ import type { Project } from "./project.model";
     {category.name}
   </button>
   {#if category.sub && category.sub.length}
-    <div class={category.isOpen ? 'visible' : 'in-visible'}>
+    <div class={category.isOpen ? 'visible' : 'hidden'}>
```

Comment:
here you could do some svelte syntax sugar: `class:visible={} class:hidden={}`
see e.g. here: https://github.com/dasch-swiss/dasch-service-platform/blob/0c176f4a37586ea25776b1eee1fb810c1aa44270/services/metadata/frontend/Header.svelte#L56

Reply:
one name it syntax sugar, another salt ;)

Reply:
shorter isn't always better, the longer syntax might be more readable. But feel free to leave it in, just wanted to point it out.

Reply:
You're right. In this case I find my version also more readable. But it's matter of habits probably.